docs/pipeline-components-and-applications/loaders-storage-targets/bigquery-loader/_cross-cloud-diagram.md

```diff
@@ -0,0 +1,18 @@
+```mdx-code-block
+import Admonition from '@theme/Admonition';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import LoaderDiagram from '@site/docs/pipeline-components-and-applications/loaders-storage-targets/bigquery-loader/_diagram.md';
+```
+
+<Tabs groupId="cloud" queryString lazy>
+  <TabItem value="aws" label="AWS">
+    <LoaderDiagram {...props} stream="Kinesis" cloud="AWS"/>
+  </TabItem>
+  <TabItem value="gcp" label="GCP">
+    <LoaderDiagram {...props} stream="Pub/Sub" cloud="GCP"/>
+  </TabItem>
+  <TabItem value="azure" label="Azure">
```

Comment:
Do we actually have an Azure artifact?

Reply:
Does `snowplow/bigquery-loader-kafka` count as Azure artifact? 😁